### PR TITLE
Hero section

### PR DESCRIPTION
**Pull request by Ravyar19**

### CODE REVIEW

**Review by murtaja1 — APPROVED** [review]

**Review by Othmanosx — APPROVED** [review]

File: src/components/HeroSection/HeroSection.jsx [hunk 1]
```diff
@@ -1,72 +1,90 @@
...
+  return (
+    <Section>
+      <Container>
+        <ColumnLeft>
+          <motion.h1
+            initial={{ opacity: 0 }}
+            animate={{ opacity: 1 }}
+            transition={{ duration: 1 }}
+          >
+            Welcome to FindYours
```
Comment (MuhannedNoman) [inline src/components/HeroSection/HeroSection.jsx:23]: Any text should be using i18next translation! 
ex: t('Welcome to FindYours')
This way translation will be easier!

File: src/components/HeroSection/HeroSection.jsx [hunk 2]
```diff
@@ -1,72 +1,90 @@
...
+          />
+          {/* <Image
+				src={PlanetFour}
+				alt='planet'
+				whileTap={{ scale: 0.9 }}
+				drag={true}
+				dragConstraints={{ left: 0, right: 0, top: 0, bottom: 0 }}
+				initial={{ opacity: 0, y: 100 }}
+				animate={{ opacity: 1, y: 0, transition: { duration: 1 } }}
+			  /> */}
```
Comment (MuhannedNoman) [inline src/components/HeroSection/HeroSection.jsx:85]: Commented code should be removed! 
Also, try enforcing this using eslint!

**Review by MuhannedNoman — CHANGES_REQUESTED** [review]

**Comment by MuhannedNoman** [comment]
Hey @Ravyar19 , nice work you did here! I left you some comments check them out and let me know what you think!

**Review by maqalaqil — APPROVED** [review]
LGTM

**Pull request merged**